### PR TITLE
Asset Management 

### PR DESCRIPTION
**Pull request by sweanan**
Closes [148](https://github.com/microsoft/dstoolkit-mlops-v2/issues/148)

Details:

Added documentation for the management/cleanup of assets such as DataAssets, Jobs, and registered Models 
Added a new sample Workflow -- cleanup registered models, which is for deleting the registred models using python sdk( this is a basic workflow )

Testing Details:

GetModelARtifacts:

https://github.com/crew-athena-org/dstoolkit-mlops-v2/actions/runs/10723157390/job/29735801612

image

DeleteRegistered Models:

https://github.com/crew-athena-org/dstoolkit-mlops-v2/actions/runs/10723211356/job/29735979508

image

### CODE REVIEW

File: docs/how-to/ModelFactoryAssetMgmt.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Context and Scope
+
+The Model Factory repository is home to machine learning models that meet metric targets and that have been promoted. The Model Factory implements engineering fundamentals to drive production quality model development. The inputs to the Model Factory are the registered datasets that are used in the CI and CD GitHub workflows to validate performance metrics published to MLFlow / Azure Machine Learning Studio through the AML Jobs(experiments). The output of the Model Factory are the registered model artifacts contained in the Azure Machine Learning Model Registry.
+
+## Goals
+
+This document outlines the process of cleanup of assets in the Azure ML workspace. These assets include Data Assets, Experiments(Jobs), and Registered Models in the Azure Machine Learning Studio that are created as part of the AML jobs.
+
+## Overview
```
Comment (marshallbentley) [inline docs/how-to/ModelFactoryAssetMgmt.md:11]: This section is blank.

File: docs/how-to/ModelFactoryAssetMgmt.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### Deleting Registered Models Artifacts
+
+The registered models in the Azure ML workspace are the model artifacts that are created as part of the CI and CD GitHub workflows.
+
+The registered Model can be deleted in 2 ways either from the Azure Machine Learning Studio Manually or using the Azure Python SDK.
+
+#### Manually Delete Model Registration
+
+To delete a registered model from the Azure Machine Learning Studio :
```
Comment (marshallbentley) [inline docs/how-to/ModelFactoryAssetMgmt.md:53]: Extra space before colon.

**Review by marshallbentley — APPROVED** [review]

File: docs/how-to/ModelFactoryAssetMgmt.md (hunk 2 above)
Comment (sweanan) [reply docs/how-to/ModelFactoryAssetMgmt.md]: Thanks @marshallbentley cleaned up

File: docs/how-to/ModelFactoryAssetMgmt.md (hunk 1 above)
Comment (sweanan) [reply docs/how-to/ModelFactoryAssetMgmt.md]: Thanks @marshallbentley cleaned up

**Pull request closed without merging**